### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by AlinaMyronenko**
in the nav tag, fails the test
please explain how to write correctly

### CODE REVIEW

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
please provide demo link and report link

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+  align-items: center;
+  max-width: 1200px;
+  background: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 40px;
+}
+
+.logo {
+  order: 1;
+  position: absolute;
```
Comment (OleziO) [inline src/style.css:21]: You can create logo without absolute position.

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+
+.nav {
+  margin: 0;
+}
+
+.nav-list {
+  height: 58px;
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
```
Comment (OleziO) [inline src/style.css:37]:
```suggestion
```

It is default value

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,89 @@
...
+}
+
+.link {
+  padding: 23px 0;
+  margin: 0 10px;
+  flex-shrink: 0;
+  align-self: center;
+  text-decoration: none;
+  position: relative;
+  font-family: "Roboto", sans-serif;
```
Comment (OleziO) [inline src/style.css:50]: Move to the body or html selector and remove it from all other places.

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,89 @@
...
+  font-size: 12px;
+  font-style: normal;
```
Comment (OleziO) [inline src/style.css:52]:
```suggestion
```

Default value

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,89 @@
...
+    left: 509px;
+  }
+}
+
+.link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
```
Comment (OleziO) [inline src/style.css:76]: Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by OleziO — CHANGES_REQUESTED** [review]
Ask the question with test in the `fe_chat`

**Pull request closed without merging**